test/UniversalRewardsDistributorTest.sol

```diff
@@ -458,6 +552,7 @@ contract UniversalRewardsDistributorTest is Test {
     }
 
     function testClaimShouldReturnsTheAmountClaimed(uint256 claimable) public {
+        vm.assume(claimable > 0);
```

Comment:
`claimable` is being bound the line below so this line is discarding test cases for no reason?